src/vs/workbench/services/positronConnections/browser/positronConnectionsService.ts

```diff
@@ -22,17 +21,13 @@ import { INotificationService, Severity } from 'vs/platform/notification/common/
 
 class PositronConnectionsService extends Disposable implements IPositronConnectionsService {
 
-	private readonly _cache: PositronConnectionsCache;
 	readonly _serviceBrand: undefined;
 
-	private onDidChangeEntriesEmitter = new Emitter<IPositronConnectionEntry[]>;
-	onDidChangeEntries: Event<IPositronConnectionEntry[]> = this.onDidChangeEntriesEmitter.event;
+	private onDidChangeConnectionsEmitter = new Emitter<IPositronConnectionInstance[]>;
```

Comment:
```
private onDidChangeConnectionsEmitter = this._register(new Emitter<IPositronConnectionInstance[]>);
```

Reply:
Resolved in fe6a0d009ddab8ff0dc5ac6b3cfd3145ec8cdd97